doc/getting_and_using_mir.md

```diff
@@ -1,157 +1,127 @@
 Getting and Using Mir  {#getting_and_using_mir}
 =====================
 
-If you are not running Ubuntu things may not work as described below. While some
-applications are able to use Mir's Wayland support only Ubuntu carries "native"
-Mir support for GTK+, SDL2 and Qt applications and the Xmir X11 server. 
+Mir is a library for building things, not an end-user product, but it does come
+with some demos to illustrate the possibilities.
 
-Mir was initially developed on and for Ubuntu. It is also available on Fedora. 
-We aim to make it available for other distributions, but currently you need to
-build it yourself. (See \ref getting_involved_in_mir).
+Getting Mir demos
+-----------------
 
-## Getting Mir examples on Ubuntu
+The Mir libraries and demos are available on Ubuntu, Fedora and Arch. It has 
+also been built and tested on Debian but, at the time or writing, is not in the
+archive.
+
+For Linux distributions that don't currently package Mir you need to build it
+yourself. (See \ref getting_involved_in_mir).
+
+## Getting Mir on Ubuntu
 
 On Ubuntu the latest Mir release is available from the Mir team's "release PPA".
 We recommend always using the latest release, but if you don't wish to use the
-PPA, there will be an earlier release of Mir in the Ubuntu archive, but some
-things may not work as described here.
+PPA, there will be also a release of Mir in the Ubuntu archive, but some things
+may not work as described here.
```

Comment:
This sentence could use a few editing passes. Before I fix it though, I have some questions on the content:
* What is the "Ubuntu archive"? Does that mean the standard Ubuntu repos?
* Why is there an outdated version of Mir anywhere? Can't we make sure the latest release gets in all the repos? we do know people at Canonical, after all.
* What does "may not work" mean exactly? Does that just mean it's an outdated version of Mir, because it sounds to me like its implying it will be more broken then that.

Reply:
>    What is the "Ubuntu archive"? Does that mean the standard Ubuntu repos?

Yes, the Ubuntu Archive Repositories. I've seen both "archive" and "repository" used.

>    Why is there an outdated version of Mir anywhere? Can't we make sure the latest release gets in all the repos? we do know people at Canonical, after all.

Because that's not how Ubuntu works. Normally only the current development release is updated, earlier series only receive bugfixes: an "exception" is needed to update earlier repo. So, bearing in mind that the release version of Mir is 0.32.1 this is what is currently in the Ubuntu Archive Repositories:
```
$ rmadison libmirclient9
 libmirclient9 | 0.21.0+16.04.20160330-0ubuntu1   | xenial          | amd64, arm64, armhf, i386, powerpc, ppc64el, s390x
 libmirclient9 | 0.26.3+16.04.20170605-0ubuntu1.1 | xenial-security | amd64, arm64, armhf, i386, powerpc, ppc64el, s390x
 libmirclient9 | 0.26.3+16.04.20170605-0ubuntu1.1 | xenial-updates  | amd64, arm64, armhf, i386, powerpc, ppc64el, s390x
 libmirclient9 | 0.28.0+17.10.20171011.1-0ubuntu1 | artful          | amd64, arm64, armhf, i386, ppc64el, s390x
 libmirclient9 | 0.31.1-0ubuntu1                  | bionic          | amd64, arm64, armhf, i386, ppc64el, s390x
 libmirclient9 | 0.31.1-0ubuntu1                  | cosmic          | amd64, arm64, armhf, i386, ppc64el, s390x
```
>    What does "may not work" mean exactly? Does that just mean it's an outdated version of Mir, because to me it implies it will be more broken then that.

The phrase is "may not work as described here" which means that something different happens. For example, prior to 0.32.0 using `miral-app` from a VT would give an error.